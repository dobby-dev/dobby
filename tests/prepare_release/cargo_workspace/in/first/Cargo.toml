[package]
name = "first-package"
version = "1.0.0"
