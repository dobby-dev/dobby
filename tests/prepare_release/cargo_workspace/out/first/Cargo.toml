[package]
name = "first-package"
version = "1.1.0"
