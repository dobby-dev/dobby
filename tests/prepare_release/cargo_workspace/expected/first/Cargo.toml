[package]
name = "first-package"
version = "1.2.0"
